.github/workflows/rhbop_productized_pull_request.yml

```
@@ -0,0 +1,56 @@
+# Tests RHBOP productized profile builds on PRs
+name: Productized Build Chain
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, labeled]
+    branches:
+      - main
+      - 8.*
+    paths-ignore:
+      - 'LICENSE*'
+      - '.gitignore'
```

according to https://github.com/kiegroup/optaplanner-quickstarts/blob/development/.github/workflows/pull_request.yml#L10
```suggestion
      - '.gitignore'
      - '.gitattributes'
```